packages/utils/package.json

```diff
@@ -56,7 +56,8 @@
     "fast-deep-equal": "^3.1.3",
     "rfdc": "^1.3.0",
     "semver": "^7.3.7",
-    "ses": "^0.15.17"
+    "ses": "^0.15.17",
+    "superstruct": "^0.16.0"
```

Comment:
We should make sure this runs in the extension fine.

Related: https://github.com/MetaMask/utils/pull/25